### PR TITLE
Correcting problem that never finds "my-default.cnf"

### PR DESCRIPTION
**Pull request by jrdalpra**
- getting "my-default.cnf" from $DIST/share folder
- ignoring failures at Integrations Tests because parallel test is not working
- ignoring souce validation at target folder

### CODE REVIEW

**Comment by dmarkov** [comment]
@jrdalpra Thanks, let me find someone who can review this pull request

**Comment by dmarkov** [comment]
@yegor256 the PR is rather short, please review it

**Comment by jrdalpra** [comment]
@dmarkov @yegor256 it's not ready. I'm just testing if CI is working fine.
I'll let you know when it's ready. Thanks.

2016-01-28 9:00 GMT-02:00 Dan Markov <notifications@github.com>:

> @yegor256 <https://github.com/yegor256> the PR is rather short, please
> review it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jcabi/jcabi-mysql-maven-plugin/pull/80#issuecomment-176123976>
> .
>



-- 
José Volmei Dal Prá Junior - (49) 9112 9359
Bacharel em Ciência da Computação - UnoChapecó
Pós-Graduado em Desenvolvimento de Software Com Java - Unoesc Xxe
Consultor Java e TotalCross
https://github.com/jrdalpra
http://www.kombo.com.br/meucurriculo/jrdalpra
http://www.linkedin.com/in/jrdalpra
msn: jr_dalpra@hotmail.com
skype: jrdalpra
e-mail: jrdalpra@gmail.com

**Comment by dmarkov** [comment]
> @dmarkov @yegor256 it's not ready. I'm just testing if CI is working fine.
> I'll let you know when it's ready. Thanks.
> 

@jrdalpra OK

**Comment by jrdalpra** [comment]
@yegor256 @dmarkov Now you can review the PR. Thanks.

**Comment by yegor256** [comment]
@jrdalpra there is definitely something wrong with pom.xml formatting. it looks ugly now. what happened? try the latest version of qulice

**Comment by jrdalpra** [comment]
@yegor256 xml is better now?

**Comment by jrdalpra** [comment]
@yegor256 can you review? I'll not use the latest qulice version because I'm getting some errors that is not part of the issue. I can create an issue to do this. Thanks.

**Comment by jrdalpra** [comment]
@dmarkov @yegor256 Should I remove this PR? Or you can review it?

**Comment by yegor256** [comment]
@jrdalpra I didn't understand, why changing pom.xml at all? leave it as is, it should work fine with the latest version of qulice

**Comment by jrdalpra** [comment]
@yegor256 My Eclipse is trying to beat me. :) The new version of qulice is complaing about @since at javadoc. A real mess.

**Comment by yegor256** [comment]
@jrdalpra just add that `since` tag and you'll be fine

**Comment by jrdalpra** [comment]
@yegor256 done

**Comment by yegor256** [comment]
@jrdalpra I still don't like that you're completely reformatting `pom.xml`. can't understand what for

**Comment by jrdalpra** [comment]
@yegor256 done. Can we merge now?

**Comment by jrdalpra** [comment]
@yegor256 can we merge? thanks.

File: pom.xml [hunk 1]
```diff
@@ -305,6 +305,20 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+              <artifactId>maven-invoker-plugin</artifactId>
```
Comment (yegor256) [inline pom.xml]: @jrdalpra I believe, we're using four spaces for indentation, in the entire document. didn't you see it?

File: pom.xml [hunk 2]
```diff
@@ -315,13 +329,16 @@
...
+                        <version>0.16.1</version>
                         <configuration>
-                            <excludes>
-                                <exclude>findbugs:.*</exclude>
-                                <exclude>duplicatefinder:.*</exclude>
-                                <exclude>pmd:/src/it/.*</exclude>
-                                <exclude>xml:/src/it/settings.xml</exclude>
-                            </excludes>
+                          <excludes>
+                            <exclude>findbugs:.*</exclude>
```
Comment (yegor256) [inline pom.xml]: @jrdalpra same here, 4 spaces please

**Comment by yegor256** [comment]
@jrdalpra see my comments

**Comment by jrdalpra** [comment]
@yegor256 done. all right now?

**Comment by jrdalpra** [comment]
@yegor256 can you review again, please?

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by jrdalpra** [comment]
@yegor256 thanks.

**Comment by dmarkov** [comment]
@yegor256 I just added **23 mins** to your account, many thanks for your contribution (000-d8b13b9c).. 765 hours and 54 mins spent here; you're getting extra minutes here (c=8); +23 to your rating, your total score is [+1649](http://www.netbout.com/b/35008?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**